website/docs/reference/network-view.mdx

```diff
@@ -8,76 +8,59 @@ title: Network View
+The **Overview** section displays the Unleash instance and applications that have connected to it within the last five minutes. Unknown applications are excluded from this view.
 
-### "unknown" applications
+Each application in the diagram includes its name and the average number of requests per second over the last five minutes.
 
-Requests that come from Unleash SDKs and other official Unleash applications will always have an application name defined. But you might sometimes see some applications listed as "unknown" in the diagrams.
+## Traffic
 
-This happens when Unleash receives requests that don't contain an application name header (`UNLEASH_APPNAME`). This can happen, for instance, if you make HTTP requests from the command line to test Unleash connections or if you write your own HTTP client for Unleash that doesn't provide an application name.
+The **Traffic** section presents the requests that have consumed the most network traffic in the last six hours. These requests are grouped by client and base URL, with up to ten groups displayed.
 
-Because Unleash can't separate these based on their application names, all "unknown" clients will get lumped together as one application in the overview.
+Unleash aggregates requests by application name and base URL. Base URLs are batched using the first two path segments after `/api`. This means that Admin API requests are separate from Client API requests.
 
-## Network overview
+## Connected Edges 
 
-The network overview is a diagram that shows the Unleash instance and known [applications](#applications) that have connected to it within the last five minutes. [Unknown applications](#unknown-applications) are not shown.
-
-Each application shown on the diagram has:
-- An application name
-- the average number of requests per second (_req/s_) that we have registered over the last five minutes.
-
-![Network overview chart](/img/network-overview.png)
-
-## Network traffic
-
-The network traffic diagram is a line chart that presents requests that have used the most network traffic over the last six hours, grouped by client and base URL for the request. For legibility, this chart only shows the ten groups that have caused the most traffic over the last six hours.
-
-Unleash aggregates requests by **client** (using application name) and **base URL**. Base URLs are batched together using the **first two** path segments following the `/api` part of the URL. In essence, that means:
-
-1. Separate requests by API: Admin API requests are separate from client API requests.
-2. Within each of these groups, group all requests by their next URL path segment. For instance: `/client/features` and `/client/features/feature-a` are grouped together, while `/client/register` and `/admin/features` are separate groups.
+:::note Availability
 
+[Enterprise](https://www.getunleash.io/pricing) | **Version**: `6.8+`
 
-![Network traffic chart](/img/network-traffic.png)
```

Comment:
I think I will add some updated screenshots back in a follow-up PR so we can get live for the 6.8 release